### ISSUE TITLE
Switching from Superset UI to Airflow UI (or reverse) forces the user to log back in

### ISSUE THREAD

**Issue by MattTriano** [MattTriano/analytics_data_where_house#83]
This is relevant to the state as of [this commit](https://github.com/MattTriano/analytics_data_where_house/commit/820620fef48578d96cbacd14ce8262c20fddf5cc), and I'm going to use this issue to log notes and hopefully the config change needed to resolve this undesirable behavior.

**Comment by MattTriano** [comment]
Based on this bit in [Airflow config documentation](https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#secret-key), coupled with the fact that both Airflow and Superset use flask for web serving and that I set a secret key in the Superset configs, I think I'll have to use the same secret key in the Airflow webserver container.

**Comment by MattTriano** [comment]
Steps taken:
* added the superset `SECRET_KEY` (as the  `AIRFLOW__WEBSERVER__SECRET_KEY` env var) and  `AIRFLOW__WEBSERVER__SESSION_BACKEND=securecookie` to the airflow .env file,
* added `PYTHONPATH=/app/pythonpath:/app/docker/pythonpath_dev` to the .env.superset file,
* removed the ss_db and ss_core volumes (although I probably could have worked more to just rotate keys from the default to the one I intended; as I hadn't included the PYTHONPATH before, I don't think the previous SECRET_KEY I set was ever used), and
* added more requirements to the airflow_requirements (and cut the py-utils container to nearly nothing) and rebuilt images.

I started the system back up and it seems the issue is solved. I'm not sure if the webserver_session config is necessary (or, I guess, I'm not sure if all of the above actions are necessary). Also, as I deleted the ss_core and ss_db volumes, I had to recreate the connection to the data-warehouse-db and some charts. But this is resolved.
